cocos/gfx/webgpu/webgpu-commands.ts

```diff
@@ -0,0 +1,1707 @@
+        case Format.RGBA8I: return 'rgba8sint';
+        case Format.R16I: return 'r16sint';
+        case Format.R16UI: return 'r16uint';
+        case Format.R16F: return 'r16sint';
+        case Format.RG16I: return 'rg16sint';
+        case Format.RG16UI: return 'rg16uint';
+        case Format.RG16F: return 'rg16float';
+        case Format.RGBA16I: return 'rgba16sint';
+        case Format.RGBA16UI: return 'rgba16uint';
+        case Format.RGBA16F: return 'rgba16float';
+        case Format.R32I: return 'r32sint';
+        case Format.R32UI: return 'r32uint';
+        case Format.R32F: return 'r32float';
+        case Format.RG32I: return 'rg32sint';
+        case Format.RG32UI: return 'rg32uint';
+        case Format.RG32F: return 'rg32float';
+        case Format.RGBA32I: return 'rgba32sint';
+        case Format.RGBA32UI: return 'rgba32uint';
+        case Format.RGBA32F: return 'rgba32float';
+        case Format.RGB10A2: return 'rgb10a2unorm';
+
+        case Format.DEPTH: return 'depth24plus';
+        case Format.DEPTH_STENCIL: return 'depth24plus-stencil8';
+
+        case Format.BC1_ALPHA: return 'bc1-rgba-unorm';
+        case Format.BC1_SRGB_ALPHA: return 'bc1-rgba-unorm-srgb';
+        case Format.BC2: return 'bc2-rgba-unorm';
+        case Format.BC2_SRGB: return 'bc2-rgba-unorm-srgb';
+        case Format.BC3: return 'bc3-rgba-unorm';
+        case Format.BC3_SRGB: return 'bc3-rgba-unorm-srgb';
+        case Format.BC4_SNORM: return 'bc4-r-snorm';
+        case Format.BC6H_SF16: return 'bc6h-rgb-float';
+        case Format.BC6H_UF16: return 'bc6h-rgb-ufloat';
+        case Format.BC7: return 'bc7-rgba-unorm';
+        case Format.BC7_SRGB: return 'bc7-rgba-unorm-srgb';
+
+        default: {
+            console.info('Unsupported Format, return bgra8unorm indefault.');
+            return 'bgra8unorm';
+        }
+    }
+}
+
+export function GFXFormatToWGPUFormat (format: Format): GPUTextureFormat {
+    return GFXFormatToWGPUTextureFormat(format);
+}
+
+function wGPUTextureFormatToGFXFormat (format: GPUTextureFormat): Format {
+    switch(format) {
+        case 'r8unorm': return Format.R8;
+        case 'r8snorm': return Format.R8SN;
+        case 'r8uint': return Format.R8UI;
+        case 'r8sint': return Format.R8I;
+        case 'rg8unorm': return Format.RG8;
+        case 'rg8snorm': return Format.RG8SN;
+        case 'rg8uint': return Format.RG8UI;
+        case 'rg8sint': return Format.RG8I;
+        case 'bgra8unorm': return Format.BGRA8;
+        case 'rgba8unorm': return Format.RGBA8;
+        case 'rgba8unorm-srgb': return Format.SRGB8_A8;
+        case 'rgba8snorm': return Format.RGBA8SN;
+        case 'rgba8uint': return Format.RGBA8UI;
+        case 'rgba8sint': return Format.RGBA8I;
+        case 'r16sint': return Format.R16I;
+        case 'r16uint': return Format.R16UI;
+        case 'r16float': return Format.R16F; // Handle potential mismatch between GFXFormat and WGPU
```

Comment:
Can you explain this mismatch? Maybe an external link can be added.

Reply:
Conversion between Cocos Gfx format and WebGPU format.

Reply:
This should be fine. I forgot to remove the comment.

Reply:
I see. The comment can be removed.